packages/calcite-components/src/components/panel/panel.tsx

```diff
@@ -614,10 +614,12 @@ export class Panel
     );
 
     return (
-      <InteractiveContainer disabled={disabled}>
-        {loading ? <calcite-scrim loading={loading} /> : null}
-        {panelNode}
-      </InteractiveContainer>
+      <Host onKeyDown={panelKeyDownHandler}>
```

Comment:
> Moves keydown handler to the host so it can be properly cancelled

For my own understanding, does this fix the issue because it allows the slotted content to handle and cancel the event before it bubbles up to the host?

Reply:
> For my own understanding, does this fix the issue because it allows the slotted content to handle and cancel the event before it bubbles up to the host?

Correct. Previously, since the event was internal, a user couldn't cancel it. It already happens by the time it bubbles up.

Reply:
![dory-gi-fs-finding-nemo](https://github.com/Esri/calcite-design-system/assets/197440/48c19ad7-a309-476e-aed2-1d3358815fe0)

Thanks for clarifying!

Reply:
cc @Elijbet